dialogue-core/src/main/java/com/palantir/dialogue/core/QueuedChannel.java

```diff
@@ -118,6 +140,8 @@ static QueuedChannel create(Config cf, LimitedChannel delegate) {
             return Optional.empty();
         }
 
+        queueUsed = true;
```

Comment:
I think i'd call this `shouldRecordQueuedTimeMetric` just to emphasise that this is purely kept around for observability purposes and shouldn't affect behaviour

Reply:
Updated to `shouldRecordQueueMetrics` since it applies to the time and counter